css/homepage.css

```diff
@@ -0,0 +1,125 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap");
+
+*,
```

Comment:
- [ ] Nice on here...Everything seems cool, but one more thing to change is the hamburger color, you are using the blue one while on the correct design, it is dark gray if I see it fine. please check this and make the necessary changes